sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -283,8 +286,9 @@ def receive(self, max_batch_size=None, max_wait_time=None):
         # type: (int, float) -> List[ReceivedMessage]
         """Receive a batch of messages at once.
 
-        This approach it optimal if you wish to process multiple messages simultaneously. Note that the
-        number of messages retrieved in a single batch will be dependent on
+        This approach it optimal if you wish to process multiple messages simultaneously, or 
```

Comment:
```suggestion
        This approach is optimal if you wish to process multiple messages simultaneously, or 
```

Reply:
So because I make bad life choices, after committing your first suggestion I THEN learned about "Add change to batch", but only after invalidating these other suggestions. (Thus copying into my reply to reapply)  Did want to say though, so appreciated that you wrote them using the suggestion feature in the first place to make this whole process easier.